e2e-tests/trade-finance-manager/cypress/integration/journeys/case/deal-page-GEF.spec.js

```diff
@@ -144,6 +151,13 @@ context('User can view a GEF AIN case deal', () => {
   });
 
   describe('facilities table', () => {
+    it('facility coverEndDate should be a dash', () => {
+      const facilityId = dealFacilities._id;
+      const facilityRow = pages.caseDealPage.dealFacilitiesTable.row(facilityId);
+
+      facilityRow.facilityEndDate().contains('-');
+    });
+
```

Comment:
For these 'should be a dash' tests, can we make it clear that "when date doesn't exist ...it should render a dash"

Reply:
done